### PR TITLE
[ImageView] Refinements

### PR DESCRIPTION
**Pull request by alloy**
- Only show placeholder color if the image needs downloading. Otherwise you might get to see the color for a very short period, which isn’t the nicest.
- Fade image in.

### CODE REVIEW

File: Pod/Classes/OpaqueImageViewComponent/AROpaqueImageView.m [hunk 1]
```diff
@@ -57,7 +57,12 @@ - (void)setImage:(UIImage *)image;
 {
   // This will cancel an in-flight download operation, if one exists.
   self.imageURL = nil;
-  [super setImage:image];
+
+  [UIView transitionWithView:self
+                    duration:0.25
+                     options:UIViewAnimationOptionTransitionCrossDissolve
+                  animations:^{ [super setImage:image]; }
+                  completion:nil];
```
Comment (orta) [inline Pod/Classes/OpaqueImageViewComponent/AROpaqueImageView.m]: this would make it tough to write tests against I assume

Comment (alloy) [inline Pod/Classes/OpaqueImageViewComponent/AROpaqueImageView.m]: Could probably be done with `[UIView setAnimationsEnabled:NO];` ?

Comment (orta) [inline Pod/Classes/OpaqueImageViewComponent/AROpaqueImageView.m]: It could, aye, guess I'm used to DI over global state, but this seem 👍

**Comment by orta** [comment]
Cool

**Pull request merged**